examples/proc_nutation.py

```diff
@@ -0,0 +1,102 @@
+"""
+=====================
+Process nutation data
+====================
+`py proc_nutation.py NODENAME FILENAME EXP_TYPE`
+
+Fourier transforms (and any needed data corrections for older data) are performed according to the `postproc_type` attribute of the data node.
+This script plots the result, as well as signal that's averaged along the `nScans` dimension.
+
+Tested with:
+
+``py proc_nutation.py nutation_1 240805_amp0p1_27mM_TEMPOL_nutation.h5 ODNP_NMR_comp/nutation``
+"""
+import pyspecdata as psd
+import pyspecProcScripts as prscr
+import numpy as np
+import matplotlib.pyplot as plt
+import sympy as sp
+import sys
+
+signal_range = (-250, 250)
+assert len(sys.argv) == 4
+s = psd.find_file(
+    sys.argv[2],
+    exp_type=sys.argv[3],
+    expno=sys.argv[1],
+    lookup=prscr.lookup_table,
+)
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3)
+with psd.figlist_var() as fl:
+    # {{{ set up subplots
+    fl.next("Raw Data with averaged scans", fig=fig)
+    fig.suptitle("Nutation %s" % sys.argv[2])
+    # }}}
+    signal_pathway = s.get_prop("coherence_pathway")
+    if "nScans" in s.dimlabels:
+        s.mean("nScans")
+    mysgn = prscr.determine_sign(
+        prscr.select_pathway(s["t2":signal_range], signal_pathway)
+    )
+    # {{{ apply overall zeroth order correction
+    s /= prscr.zeroth_order_ph(prscr.select_pathway(s["t2":0], signal_pathway))
+    d_raw = s.C
+    fl.image(
+        prscr.select_pathway(s["t2":signal_range], signal_pathway), ax=ax1
+    )
+    ax1.set_title("Raw Data")
+    d_unc = prscr.select_pathway(
+        s["t2":signal_range].real, signal_pathway
+    ).integrate("t2")
+    # }}}
+    # {{{ zeroth_order on individual betas
+    for j in range(len(s.getaxis("beta"))):
+        ph0 = prscr.zeroth_order_ph(
+            prscr.select_pathway(
+                s["beta", j]["t2":signal_range], signal_pathway
+            )
+        )
+        s["beta", j] /= ph0
+    d0 = prscr.select_pathway(
+        s["t2":signal_range].real, signal_pathway
+    ).integrate("t2")
+    # }}}
+    # {{{ define mysgn based on phase difference between d_unc and d0
+    mysign = (d0 / d_unc).angle / np.pi
+    mysign = np.exp(1j * np.pi * mysign.run(np.round))
```

Comment:
why not just use all the above to overwrite the existing function that determines the sign?

Reply:
also, why is this used for the 2 pulse variant, but  not for the one pulse variant?

Reply:
I removed this mysign stuff as I'm fairly sure it was diagnostic and what I have now works well just using the fid_from_echo. I agree with overwriting the determine_sign but that should be a separate PR I think?

Reply:
this was not the case and mysign / looking at phase variation was added back in